buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -19,6 +19,6 @@ object Apps {
     const val targetSdk = 29
     const val versionCode = 1
     const val versionName = "1.0"
-    const val packageName = "com.game.quillyjumper"
+    const val packageName = "com.github.quillraven.quillyjumper"
     const val name = "Quilly Jumper"
```

Comment:
please also rename the `name` to Quilly's Adventure

Reply:
Reading stream chat helps my friend